gui/default/syncthing/device/changeLogModalView.html

```diff
@@ -0,0 +1,26 @@
+<style> th, td { padding: 6px; } </style>
+<modal id="globalDiskChanges" status="default" icon="{{'history'}}" heading="{{'Global Changes' | translate}}" large="yes" closeable="yes">
+    <div class="modal-body">
+        <table>
+          <tr>
+            <th translate>Device</th>
+            <th translate>Action</th>
+            <th translate>Type</th>
+            <th translate>Path</th>
+            <th translate>Time</th>
+          </tr>
+          <tr ng-repeat="changeEvent in globalChangeEvents">
+            <td>{{deviceName(findDevice(changeEvent.data.modifiedBy))}}</td>
```

Comment:
Ok so I just changed this back to short ID and it broke predictably.  This line here needs the full device ID and not a short ID.

I want this to work in a way that satisfies you so I can get this PR'ed.  Any chance you can help me find the function I'm looking for in javascript to search for the full ID using the short?  I'm looking here but coming up blank.